lib/html/includes.tmpl

```diff
@@ -61,6 +61,15 @@
               </td>
             {{/each}}
             </tr>
+            {{if activerecord_query_cache_hits.length > 0}}
+              <tr>
+                <td colspan="3">&nbsp;</td>
+                <td colspan="2" class="profiler-number profiler-percent-in-sql" title="${activerecord_query_cache_hits} queries hit the cache">
```

Comment:
my preferred wording here would be "served by ActiveRecord::QueryCache", idk if that fits/is too lengthy.